packages/auth-server/components/views/Auth.vue

```diff
@@ -29,7 +29,7 @@
       </p>
     </CommonHeightTransition>
 
-    <div class="flex flex-col gap-5 mt-8 py-8">
+    <div class="flex flex-col gap-5 mt-8 py-8 w-[300px] m-auto">
```

Comment:
it would break for smaller than 300px screens

Reply:
You're right, added a fix